tests/simple/testdata/integer_math.textproto

```diff
@@ -1,2 +1,171 @@
 name: "integer_math"
 description: "Tests for int and uint math."
+section {
+  name: "int64_math"
+  description: "Simple tests for int64."
+  test {
+    name: "simple_add_positive_positive"
+    expr: "42 + 2"
+    value: { int64_value: 44 }
+  }
+  test {
+    name: "simple_add_positive_negative"
+    expr: "42 + (-2)"
+    value: { int64_value: 40 }
+  }
+  test {
+    name: "simple_add_negative_negative"
+    expr: "-42 + (-2)"
+    value: { int64_value: -44 }
+  }
+  test {
+    name: "simple_sub_positive_positive"
+    expr: "42 - 2"
+    value: { int64_value: 40 }
+  }
+  test {
+    name: "simple_sub_positive_negative"
+    expr: "42 - (-2)"
+    value: { int64_value: 44 }
+  }
+  test {
+    name: "simple_sub_negative_negative"
+    expr: "-42 - (-2)"
+    value: { int64_value: -40 }
+  }
+  test {
+    name: "simple_multiply_positive_positive"
+    expr: "42 * 2"
+    value: { int64_value: 84 }
+  }
+  test {
+    name: "simple_multiply_positive_negative"
+    expr: "42 * (-2)"
+    value: { int64_value: -84 }
+  }
+  test {
+    name: "simple_multiply_negative_negative"
+    expr: "-42 * (-2)"
+    value: { int64_value: 84 }
+  }
+  test {
+    name: "simple_devide_positive_positive"
+    expr: "42 / 2"
+    value: { int64_value: 21 }
+  }
+  test {
+    name: "simple_devide_positive_negative"
+    expr: "-42 / 2"
+    value: { int64_value: -21 }
+  }
+  test {
+    name: "simple_devide_negative_negative"
+    expr: "-42 / (-2)"
+    value: { int64_value: 21 }
+  }
+  test {
+    name: "simple_mod_positive_positive"
+    expr: "42 % 5"
+    value: { int64_value: 2 }
+  }
+  test {
+    name: "simple_mod_positive_negative"
+    expr: "42 % (-5)"
+    value: { int64_value: 2 }
+  }
+  test {
+    name: "simple_mod_negative_negative"
+    expr: "-42 % (-5)"
+    value: { int64_value: -2 }
+  }
+  test {
+    name: "simple_mod_zero"
+    expr: "42 % 0"
+    eval_error: {
+      errors: { message:"modulus by zero" }
+    }
+  }
+  test {
+    name: "simple_negtive"
+    expr: "-(42)"
+    value: { int64_value: -42 }
+  }
+  test {
+    name: "simple_negtive_zero"
+    expr: "-(0)"
+    value: { int64_value: 0 }
+  }
+  test {
+    name: "double_negtive"
+    expr: "-(-42)"
+    value: { int64_value: 42 }
+  }
+  test {
+    name: "devide_zero"
+    expr: "15 / 0"
+    eval_error: {
+      errors: { message:"divide by zero" }
+    }
+  }
+  test {
+    name: "multiply_zero"
+    expr: "15 * 0"
+    value: { int64_value: 0 }
+  }
```

Comment:
Add the following tests, here and in the following sections as appropriate:
- add_left_identity:  0 + 17 &rarr; 17
- add_right_identity:  83 + 0 &rarr; 83
- add_commutative:  36 + 21 == 21 + 36 &rarr; true
- add_associative: 4 + (15 + 3) == (4 + 15) + 3 &rarr; true (with a note that this is not true when intermediate results might overflow)
- mul_left_identity: 1 * 28 &rarr; 28
- mul_right_identity: -9 * 1 &rarr; -9
- mul_commutative: 39 * 72 == 72 * 39 &rarr; true
- mul_associative: 2 * (7 * -13) == (2 * 7) * -13 &rarr; true (again with a note that this is not true when intermediate results overflow)
- add_mul_distribute: 4 * (2 + 7) == 4 * 2 + 4 *7 &rarr; true